### ISSUE TITLE
AttributeError: 'PosixPath' object has no attribute 'lower'

### ISSUE THREAD

**Issue by maia** [RhetTbull/osxphotos#392]
When running: 

`osxphotos export ~/export --export-by-date --only-photos --skip-bursts --skip-live --jpeg-ext jpg --download-missing --skip-original-if-edited --cleanup --skip-raw --verbose`

I had the error message:

```
Cleaning up /Users/…/export
Traceback (most recent call last):
  File "cli.py", line 18, in <module>
  File "site-packages/click/core.py", line 764, in __call__
  File "site-packages/click/core.py", line 717, in main
  File "site-packages/click/core.py", line 1137, in invoke
  File "site-packages/click/core.py", line 956, in invoke
  File "site-packages/click/core.py", line 555, in invoke
  File "site-packages/click/decorators.py", line 27, in new_func
  File "site-packages/click/decorators.py", line 17, in new_func
  File "osxphotos/cli.py", line 1509, in export
  File "osxphotos/cli.py", line 3009, in cleanup_files
  File "osxphotos/cli.py", line 3009, in <dictcomp>
AttributeError: 'PosixPath' object has no attribute 'lower'
[30454] Failed to execute script cli
```

I'm running the current executable from the releases page: osxphotos_MacOS_exe_darwin_x64_v0.41.0.zip

**Comment by RhetTbull** [comment]
I think this should be fixed in [v0.41.1](https://github.com/RhetTbull/osxphotos/releases/tag/v0.41.1)  Let me know if you're still having problems.

**Comment by maia** [comment]
That solved the issue, thanks a lot! for the quick fix
